main.py

```diff
@@ -76,7 +81,10 @@ def main(
                 organization_name, repo_name = gh_repo.split("/")
                 github.get_organization(organization_name).create_repo(
                     repo_name,
-                    description=f"Migrated from Bitbucket https://bitbucket.org/{bb_repo}",
+                    description=(
+                        f"Migrated from Bitbucket https://bitbucket.org/{bb_repo}\n"
+                        f"{bitbucket_client.get_repo_description()}"
```

Comment:
A detail: I think it's best to emphasize the description by printing it first..

```suggestion
                        f"{bitbucket_client.get_repo_description()}\n\n"
                        f"Migrated from Bitbucket https://bitbucket.org/{bb_repo}"
```